hudi-utilities/src/test/java/org/apache/hudi/utilities/deltastreamer/TestHoodieDeltaStreamerWithMultiWriter.java

```diff
@@ -82,6 +83,7 @@ void testUpsertsContinuousModeWithMultipleWritersForConflicts(HoodieTableType ta
     tableBasePath = basePath + "/testtable_" + tableType;
     prepareInitialConfigs(fs(), basePath, "foo");
     TypedProperties props = prepareMultiWriterProps(fs(), basePath, propsFilePath);
+    props.setProperty(HoodieCompactionConfig.PARQUET_SMALL_FILE_LIMIT.key(), "0");
```

Comment:
should this be applicable to some cases in TestHoodieDeltaStreamer ?

Reply:
yes, I did not want to boil the ocean just yet. so just focussing on most frequently failing tests. will file a jira to track other similar tests. https://issues.apache.org/jira/browse/HUDI-5595